tests/test_inner.py

```diff
@@ -1238,19 +1238,21 @@ def restore_logger():
                                                       'watcher': watch_exit
                                                   }}],
                                    plugin_files=[this_file])
+    # BaseException repr does not include trailing comma in Python >= 3.7
+    # we look for a partial match in log strings for Python < 3.7 compatibility
+    expected_entry = "plugin '{}_watched' raised an exception:" + \
```

Comment:
pep8 (and I) discourages using line escaping in favor of parentheses and concatenation of constant strings using + is not the nicest too.
Could be this rewritten to:
```python
expected_entry = (
    "plugin '{}_watched' raised an exception: "
    "BuildCanceledException('Build was canceled'"
)
```